src/status_im/ui/screens/chat/input/input.cljs

```diff
@@ -21,8 +21,8 @@
             [status-im.utils.utils :as utils]))
 
 (defview basic-text-input [{:keys [set-container-width-fn height single-line-input?]}]
-  (letsubs [{:keys [input-text]} [:get-current-chat]
-            cooldown-enabled?    [:chat-cooldown-enabled?]]
+  (letsubs [{:keys [input-text]} [:chats/current-chat]
+            cooldown-enabled?    [:chats/cooldown-enabled?]]
```

Comment:
what is this flag used for? I know only the namespace is changing but just curious.

Reply:
it's for preventing user from sending more than n messages in a short period of time

Reply:
makes sense. thanks!